docupload/models.py

```diff
@@ -1,13 +1,16 @@
+from django.conf import settings
 from django.db import models
 
 
-# Create your models here.
-
 class Documentation(models.Model):
     '''
     Documentation: Model class which holds details of uploaded documentation file
     '''
 
     name = models.CharField(max_length=200)
-    doc_file = models.FilePathField('../static/docs/')
```

Comment:
This implementation has changed a bit in the other two PRs by @Arunabh98. Plus I dont think storing in the database is good because static html doc files will load faster, and also we will have to keep a track of all the extracted media which will be painful. We anyway need to convert it first to a file for pandoc to be able to convert it.

I dont even know why this change exists in this PR though. @CodeMaxx You need to rebase.